### PR TITLE
Advance toggle with Retract button

### PR DESCRIPTION
**Pull request by hhdtan**
advance uses an ionic toggle

retract enables/disables the toggle ^

99% ionic file installations, 1% me.

Updated height of nav-bar to reflect actual height needed for the bar. Ionic made things wonky and I have no idea how did 35px worked prior

### CODE REVIEW

**Comment by hhdtan** [comment]
image
image
image

**Review by yeeteing — COMMENTED** [review]
look good! some small changes needed though

File: OnLand/ControlLaptop/angular-UI/src/app/components/advance-retract/advance-retract.component.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div class="advance-retract-container">
+
+  <div class="advance-label">
+    ADVANCE
+    <ion-toggle class="advance-toggle" [disabled]="advanceFlag"></ion-toggle>
```
Comment (yeeteing) [inline OnLand/ControlLaptop/angular-UI/src/app/components/advance-retract/advance-retract.component.html:5]: We don't want the "toggle" to be **disable** ,when the "button" is being click. We want the "button" to be **disable**, when the "toggle" is "red".

File: OnLand/ControlLaptop/angular-UI/src/app/components/advance-retract/advance-retract.component.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  </div>  
+
+  <button class="retract-button" (click)="toggleFlag()"> RETRACT </button>
```
Comment (yeeteing) [inline OnLand/ControlLaptop/angular-UI/src/app/components/advance-retract/advance-retract.component.html:8]: So, when "ion-toggle" is **red** for example, the button would be "disabled". To do so, you can look into:
https://www.w3schools.com/tags/att_button_disabled.asp

**Review by yeeteing — COMMENTED** [review]
just some small changes~ otherwise is good

**Comment by hhdtan** [comment]
initial (advance - false, retract - disabled) 
image

toggled 
image

**Comment by colton-smith** [comment]
Looks good, just one quick note:

The retract button should be available only when the advance toggle is disabled, looks like its currently the opposite

**Comment by yeeteing** [comment]
- made changes so that the retract button is available only when the advance toggle is disabled
- added some css feature
- allow continuos firing when the mouse is being press do
Screen Shot 2021-03-07 at 2 56 36 PM
Screen Shot 2021-03-07 at 2 56 39 PM

**Review by yeeteing — APPROVED** [review]
look good!

**Pull request merged**